### ISSUE TITLE
Minor core change to extend user UI Control (edit fields)

### ISSUE THREAD

**Issue by AnthonyMuscio** [Jermolene/TiddlyWiki5#3303]
Folks,

My Second attempt to get something changed in TiddlyWiki as per the short [discussion](https://groups.google.com/forum/?hl=en#!topic/tiddlywikidev/3kEcg7Nt0VI)

To allow customisation of the editField lines when in edit mode.

I would like to propose the insertion of a template reference, perhaps named 
$:/core/ui/EditTemplate/fields/more-template (as Suggested by Mario)
in the tiddler $:/core/ui/EditTemplate/fields

This should occur as follows;
<$action-deletefield $field=<<currentField>>/>
{{$:/core/images/delete-button}}
</$button> {{||$:/core/ui/EditTemplate/fields/more-template}}

This would allow additional code to be placed to the right of the delete button on each field listed in edit mode without having to edit the core tiddler **$:/core/ui/EditTemplate/fields**

I am currently building tools for manipulating fields such as

- Create/link to a tiddler for each field name "a Field Tiddler"
- Provide a tool tip to describe the field (using description field of a field Tiddler)
- Provide a selection filter for field values
- and more

I imagine others could benefit from this "hook" into an important part of the UI, this seems a simple change to extend customisation and to preserve core tiddlers. This would open Field manipulation to a new height.

It would perhaps make sense to add to $:/core/ui/EditTemplate/fields/more-template
something like this;

<$list filter="[all[shadows+tiddlers]tag[$:/tags/fields-more]!has[draft.of]]" variable="listItem"><$transclude tiddler=<<listItem>>/></$list>

Such that we can simply tag tiddlers to appear to the right of each field listed in the edit template view. Most often tiddlers so tagged would be buttons like the existing delete button.

Please comment and advise, my names are only suggestions, tell me a better naming standard. Then I would like to create my first pull request?

Regards
Tony

**Comment by Jermolene** [comment]
Hi @AnthonyMuscio I think the consistent way to approach this would indeed be via a new system tag that is transcluded in the existing `$:/core/ui/EditTemplate/fields` tiddler.

**Comment by AnthonyMuscio** [comment]
Bump, I want to turn this into a change request?

Any additional advice before I suggest one?

**Comment by morosanuae** [comment]
@AnthonyMuscio Probably your comments are full of great ideas but to be honest with you I never read them because they are too long and/or with a dull presentation and I never have time and disposition, though I'm really interested. 
Please, don't get me wrong, I just want your ideas reaching for a wider audience (including myself). So here are my suggestions for you:
- Express a single idea in a comment/issue
- Present it as succinctly as you can! Many people are tired from work, they will skip your issues/comments if they are too long/boring.
- Present it as visually as you can. Make a demo (screenshot). An image worth a thousands words!
- Wait for feedback from others before commenting again (otherwise there will be too much info to process). I personally find it very hard to follow your issues/comments.

I know there will be an increased effort from you but I think it will not be in vain. Quality over quantity!

Thank you for you understanding and apologies!

P.S. @Jermolene I try to understand why there isn't much participation from users on GitHub as I see it in Google forums. Only a handful of people express their opinion regarding opened issues. Or am I wrong?
